utils/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/ConfluentSchemaRegistryClient.scala

```diff
@@ -29,6 +32,35 @@ trait ConfluentSchemaRegistryClient extends SchemaRegistryClient with LazyLoggin
     }
 }
 
+class DefaultConfluentSchemaRegistryClient(override val client: CSchemaRegistryClient) extends ConfluentSchemaRegistryClient {
```

Comment:
I don't think I fully understand why is this needed for this change?

Reply:
Because in tests was used cached implementation and each test case was changing behaviour of other test cases